tools/start_server.py

```diff
@@ -146,9 +147,22 @@ def handle_save_ptu_to_file(self, _method_name, _params):
     file_name = _params["fileName"]
     content = json.loads(_params["content"])
 
-    with open(file_name, 'w') as json_file:
-      json.dump(content, json_file)
+    # Validate file path
+    def isFileNameValid(_file_name):
+      path = os.path.abspath(os.path.normpath(_file_name))      
```

Comment:
from docs it seems `normpath` will return an absolute path making the `abspath` call redundant

Reply:
`os.path.normpath('ptu.json')` just returns `ptu.json` whereas abspath would return the full path which is used by the later check